src/App.tsx

```diff
@@ -1,7 +1,35 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string;
+};
+
+export class App extends React.Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handler);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handler);
+  }
+
+  handler = (event: KeyboardEvent) => {
```

Comment:
Method's name should start with a verb (so you could easily tell what your method actually do)
```suggestion
  handleKeyPress = (event: KeyboardEvent) => {
```

Reply:
Let's render `space` if I clicked the Space button
<img width="490" alt="image" src="https://user-images.githubusercontent.com/32794096/208607171-b2ffa022-9084-492d-9328-ad1df21f88b4.png">